apps/studio/src/server/modules/page/page.router.ts

```diff
@@ -40,6 +41,13 @@ export const pageRouter = router({
       }
     }),
 
+  getIsomerJsonSchema: publicProcedure.query(async () => {
+    const schema = (await fetch(ISOMER_SCHEMA_URI).then((response) =>
+      response.json(),
+    )) as IsomerJsonSchema
+    return { schema }
+  }),
```

Comment:
recently i've been introduced to a library called `ky` - it seems to be the correct library to use here. this is because: 
1. `fetch` doesn't treat 4xx/5xx as errors 
2. `fetch` isn't actually stable (requires a polyfill) 
3. `fetch` doesn't retry so if the problem is ephemeral, this request will die anyway. 

alternatively, we could instead just store the json as a build artifact on backend , which is also ok to me

Reply:
We resolved to move the schema into the components package, done in 71a18021a67d22a779bab79620dc28904bb1c7b1.